dbms/src/TestUtils/tests/gtest_mock_executors.cpp

```diff
@@ -41,27 +41,29 @@ TEST_F(MockDAGRequestTest, MockTable)
 try
 {
     auto request = context.scan("test_db", "test_table").build(context);
-    String expected_string_1 = "table_scan_0\n";
-    ASSERT_DAGREQUEST_EQAUL(expected_string_1, request);
+    String expected_string = "table_scan_0 | columns:{index: 0, type: String, index: 1, type: String}\n";
+    ASSERT_DAGREQUEST_EQAUL(expected_string, request);
 
     request = context.scan("test_db", "test_table_1").build(context);
-    String expected_string_2 = "table_scan_0\n";
-    ASSERT_DAGREQUEST_EQAUL(expected_string_2, request);
+    String expected_string_1 = "table_scan_0 | columns:{index: 0, type: Long, index: 1, type: String, index: 2, type: String}\n";
+    ASSERT_DAGREQUEST_EQAUL(expected_string_1, request);
```

Comment:
```suggestion
    {
        String expected_string = "table_scan_0 | columns:{index: 0, type: String, index: 1, type: String}\n";
        ASSERT_DAGREQUEST_EQAUL(expected_string, request);
    }
    request = context.scan("test_db", "test_table_1").build(context);
    {
        String expected_string = "table_scan_0 | columns:{index: 0, type: Long, index: 1, type: String, index: 2, type: String}\n";
        ASSERT_DAGREQUEST_EQAUL(expected_string, request);
    }
```
How about writing in this way?